kube/save-return-lookup/deployment.yml

```diff
@@ -140,7 +140,11 @@ spec:
             - name: PROXY_CLIENT_SECRET
               valueFrom:
                 secretKeyRef:
+                  {{ if eq .KUBE_NAMESPACE .PROD_ENV }}
+                  name: keycloak-secret-lookup
+                  {{ else }}
                   name: keycloak-lookup-secret
+                  {{ end }}
```

Comment:
We are going to change this in the other ticket you created as prod is currently using the old keyloak url for lookup. You can revert this for now and we can change it when working on the new ticket.